pavelib/quality.py

```diff
@@ -50,8 +50,7 @@ def find_fixme(options):
         apps_list = ' '.join(top_python_dirs(system))
 
         pythonpath_prefix = (
-            "PYTHONPATH={system}:{system}/lib"
-            "common/djangoapps:common/lib".format(
+            "PYTHONPATH={system}:common/djangoapps:common/lib".format(
```

Comment:
In {system}/env/common.py, do we actually include {system} in the PYTHONPATH?  It looks like it includes {system}/djangoapps, but not just {system}.